libs/barista-components/radial-chart/src/radial-chart.html

```diff
@@ -22,7 +22,11 @@
     <dt-legend-item *ngFor="let series of _renderData">
       <dt-legend-symbol
         class="dt-radial-chart-legend-symbol"
-        [style.--dt-radial-chart-legend-symbol-color]="series.color"
+        [style]="
+          sanitizer.bypassSecurityTrustStyle(
```

Comment:
I'd suggest doing the sanitization in the class and not directly in the template. This keeps the template clean and we add a descriptive comment why we need to do this.
```
_sanitizeCustomProperty(variable: string, value: string): string {
  return this._sanitizer.bypassSecurityTrustStyle(`${variable}: ${value}`);
}
```
Please also prefix the sanitizer with an _ and make it private if we move the sanitization to the class.